iree/tools/CMakeLists.txt

```diff
@@ -478,6 +478,21 @@ if(${IREE_BUILD_COMPILER})
     RUNTIME DESTINATION bin
   )
 
+  if(NOT LLD_SYMLINKS_TO_CREATE)
+  set(LLD_SYMLINKS_TO_CREATE
+    lld-link ld.lld ld64.lld wasm-ld)
```

Comment:
These are different programs. Why add symlinks that group them together?

`wasm-ld` in particular has flags and behavior that `ld` doesn't: https://lld.llvm.org/WebAssembly.html